### PR TITLE
Enochtsang/sys 7/readme revamp

### PR DESCRIPTION
**Pull request by ExtraGravity**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+== Configuration
+
+The agriculture-core pulls its configuration from a `.env` file at the root of the project.
+These configuration variables are loaded into `process.env.<VAR_NAME>` and are accessible accross the application.
+To get started, this configuration file must be created with the following format:
+
+[source,properties]
+----
+# Application Configuration
+PORT=
```
Comment (Parasrah) [inline README.adoc:14]: # Requested Favour

This is absolutely my fault from when I made the README originally, I forgot to add a `LOG_LEVEL` option. This has the option of:
```
{ 
  error: 0, 
  warn: 1, 
  info: 2, 
  verbose: 3, 
  debug: 4, 
  silly: 5 
}
```
And will control which logs are generated

**Review by Parasrah — CHANGES_REQUESTED** [review]
Other than my own stupid blunder looks great!

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+ * verbose: 3
+ * debug: 4
+ * silly: 5
+
+== Development
+
+=== Dependencies
+
+* link:https://nodejs.org/en/[NodeJS] is the framework in use.
+* MySQL server our some other valid host indicated by the `.env` file.
```
Comment (Parasrah) [inline README.adoc]: Typo here I missed. Also, I think it requires a 1-line change in the `knex` configuration to swap out which db is used

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+=== Naming Conventions
+
+Creating a new test file is easy!
+You can add your unit tests in `<ROOT>/test/unit` and integration tests in `<ROOT>/test/integration`.
+Just make sure your files end with `.test.ts` so they will be picked up in the webpack build.
+
+Unit tests go in `agriculture-core/test/unit/`.
+Integration tests go in `agriculture-core/test/integration`.
+To ensure the files are discovered by the webpack build, test files must have the suffix `.test.ts`.
```
Comment (Parasrah) [inline README.adoc]: Did you intend to get rid of my lines above? I missed that these are repeated before

**Review by Parasrah — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 3 above)
Comment (ExtraGravity) [reply README.adoc]: Yup, did intend to do that.

File: README.adoc [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+=== Dependencies
+
+* link:https://nodejs.org/en/[NodeJS] is the framework in use.
+* link:https://yarnpkg.com/lang/en/docs/install/[yarn] handles all other dependencies.
+* MySQL server indicated by the `.env` file.
+
+To install all other dependencies with use the command `yarn`.
+
+NOTE: The `knex` configuration must also be altered to choose the right `client`.
```
Comment (Parasrah) [inline README.adoc:51]: Just mentioning this note is still here, if that was intentional that's fine :D

**Review by Parasrah — APPROVED** [review]
Looks good!

File: README.adoc (hunk 4 above)
Comment (ExtraGravity) [reply README.adoc:51]: Yup intentional

**Pull request merged**